plugins/smb/plugin.spec.yaml

```diff
@@ -106,10 +118,11 @@ actions:
         description: Message to send to the remote server
         type: string
         required: true
+        example: Hello world
     output:
       response:
-        description: Response
-        title: Server response
+        description: Server response
+        title: Response
```

Comment:
Is this necessary to change? If it is not, then we can get by with only a minor version increment on this plugin.

Reply:
It will fail TitleValidator with `Title contains a lowercase 'response' when it should not.`